blocks/shared-styles/_children/headings/section/index.jsx

```diff
@@ -3,9 +3,9 @@ import PropTypes from "prop-types";
 
 import LevelContext from "../context";
```

Comment:
Any reason for these changes? We are trying to remove the usage of these components

Reply:
As mentioned in the huddle, this change was mainly due to the linter flagging these files for review and it coming up.